core_domain.process_model_execution/src/flow_node_handler/activity_handler/call_activity_handler.ts

```diff
@@ -154,56 +174,68 @@ export class CallActivityHandler extends ActivityHandler<Model.Activities.CallAc
     identity: IIdentity,
   ): Promise<Array<Model.Base.FlowNode>> {
 
-    try {
-      this.publishActivityReachedNotification(token);
+    const handlerPromise = new Promise<Array<Model.Base.FlowNode>>(async (resolve: Function, reject: Function): Promise<void> => {
 
-      await this.persistOnSuspend(token);
+      try {
+        this.publishActivityReachedNotification(token);
 
-      const callActivityResult = await this.executeSubprocess(identity, processTokenFacade, token);
+        this.onInterruptedCallback = async (): Promise<void> => {
+          await this.processInstanceManagementService.terminateProcessInstance(identity, this.subProcessInstanceId);
```

Comment:
Hier sollte ein Null Check hin, für den Fall, dass der Interrupt passiert, bevor der Subprozess gestartet werden konnte.
In dem Fall wäre `subProcessInstanceId` undefined.

```suggestion
          if (this.subProcessInstanceId) {
            await this.processInstanceManagementService.terminateProcessInstance(identity, this.subProcessInstanceId);
          }
```